### ISSUE TITLE
Embed a tentative interactive roadmap figure on the OSO website

### ISSUE THREAD

**Issue by himalayajung** [open-science-org/admin#8]
Embed a tentative interactive roadmap figure on the OSO website. 

Below is the working prototype made by @KeithSSmith 

<!DOCTYPE html>
<html>
  <body>
 
    <h1>Test Coggle</h1>
    <iframe width='853' height='480' src='https://embed.coggle.it/diagram/WdKgt2gXiQABDP98/dcf71d806ecbc0579dcdf4160c430ba358a767e1b34476498368efab248aacda' frameborder='0' allowfullscreen></iframe>
 
  </body>
</html>

**Comment by oso-network** [comment]
@rchenmit @kdgosik @KeithSSmith @abinashk @bikos @himalayajung @canautumn
priority 1 (highest level) assigned for this issue

**Comment by himalayajung** [comment]
@abinashk @bikos @KeithSSmith @kdgosik Should we put the tentative roadmap figure (after your edits) on the website?

**Comment by abinashk** [comment]
@himalayajung yes we can put it on the website. But first lets decide on the timeline.
